sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesResultCollection.cs

```diff
@@ -8,22 +8,22 @@ namespace Azure.AI.TextAnalytics
 {
     /// <summary>
     /// </summary>
-    public class ExtractLinkedEntitiesResultCollection : ReadOnlyCollection<ExtractLinkedEntitiesResult>
+    public class RecognizePiiEntitiesResultCollection : ReadOnlyCollection<RecognizePiiEntitiesResult>
```

Comment:
Question for future API discussion - is it worth having a base ResultCollection<T> for these?

Reply:
We tried this, and Krzysztof didn't like using generics for this.  But I had the same thought.